nva-datamodel-java/src/main/resources/publication-ontology.ttl

```diff
@@ -1343,7 +1343,7 @@ nva:concertProgramme a rdf:Property ;
   rdfs:range nva:MusicalWorkPerformance .
 
 nva:publishedDate a rdf:Property ;
-  rdfs:comment "This represents meta-metadata about when the Publication description changed status to Published" ;
+  rdfs:comment "This represents meta-metadata about when the Publication description changed status to Published in this system" ;
```

Comment:
"The date that the thing was originally published for public consumption in the documenting system" avoids details